src/WebJobs.Script/Utility.cs

```diff
@@ -533,8 +533,29 @@ private static bool ContainsFunctionWithWorkerRuntime(IEnumerable<FunctionMetada
             return false;
         }
 
-        public static bool CheckAppOffline(string scriptPath)
+        public static async Task MarkContainerDisabled(ILogger logger)
         {
+            logger.LogInformation("Setting container instance offline");
```

Comment:
nit: change to logDebug and add two logs, if file exists add log, files exists and if not then, log you have makes sense

Reply:
changed to logdebug. if this is called twice we will have 2 entries for "Setting container instance offline". so we might not need additional logs for that scenario. Also this operation is idempotent and might get called multiple times due to retries on dataservice.